### DISCUSSION TITLE
What is expected with `enable host-network-access` ?

### DISCUSSION THREAD

**Discussion by matgeb83** [crc-org/crc#2291] [Q&A]
when we enable host-network-access,  does the host get TCP/IP connections  to the CRC containers? or it is just the other way round only? how would services that i run on the host machine get the ability to connect to the crc containers?

**Comment by praveenkumar** [comment]
@matgeb83 Can you please explain a bit more with all the steps, like what you want to try, what kind of service ..etc.?

**Reply by matgeb83** [reply]
@praveenkumar I have containers that are running in CRC and  on the CRC host  I got an application running trying to create TCP/IP connections to  the pod but the Pod is not reachable from the host.   With the host-network-access enabled at the time of CRC start,  it looks like the pod can reach or have a route to the host IP, but the host cannot reach the pod and thought enabling the host-network-access would provide TCP connections on both directions.

**Reply by cfergeau** [reply]
`host-network-access` is indeed about pod -> host network connections. For the other way around, `oc port-forward` may be an option?

**Comment by matgeb83** [comment]
oc port-forward  worked but i was looking for more stable way of doing it and i did it with keepalived-oprator
